Remotion/Validation/UnitTests/Merging/OrderPrecedenceValidationRuleCollectorMergerTest.cs

```diff
@@ -42,7 +42,7 @@ public class OrderPrecedenceValidationRuleCollectorMergerTest
     private Expression<Func<Customer, string>> _firstNameExpression;
     private Expression<Func<Customer, string>> _lastNameExpression;
     private NotNullValidator _notNullValidator;
-    private NotEmptyValidator _notEmptyValidator;
+    private NotEmptyValueValidator _notEmptyValueValidator;
```

Comment:
Note: many (all) of these unit and integration tests will get touched again when we update to NotEmptyOrWhitespaceValidator since we use string property validation so much in the test domain.